src/registration/ApplicationActivation.Library/ApplicationActivationService.cs

```diff
@@ -245,11 +245,14 @@ await _provisioningManager.DeleteClientRolesFromCentralUserAsync(iamUserId, role
         }
     }
 
-    private async Task SetTheme(IEnumerable<string> iamIdpAliasse)
+    private async Task SetTheme(IEnumerable<IdentityProviderForTheme> iamIdps)
     {
-        foreach (var alias in iamIdpAliasse)
+        foreach (var iamIdp in iamIdps)
```

Comment:
please change to and remove the if statement
```suggestion
        foreach (var iamIdp in iamIdps.Where(i => i.IdentityProviderType == IdentityProviderTypeId.SHARED))
```